LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2017 Évelyne Lachance
+Copyright (c) 2018 YorkAARGH
```

Comment:
The license hasn't changed, this should remain as Évelyne.

Reply:
Year 2018 is in one branch so kept the updated one

Reply:
Cool. Don't change the **author** name, thanks.

Reply:
Ok